### PR TITLE
`dex.trades`: update project name to remove spaces

### PR DESCRIPTION
**Pull request by jeff-dude**
due to downstream dependencies, spaces within `project` column values causes issues since it's a partition column. our simple request is to rename the project names.

### CODE REVIEW

File: dbt_subprojects/dex/models/_projects/oneinch/oneinch_lop_own_trades.sql [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 select
     blockchain
-    , '1inch LOP' as project
+    , '1inch-LOP' as project
```
Comment (jeff-dude) [inline dbt_subprojects/dex/models/_projects/oneinch/oneinch_lop_own_trades.sql:14]: fyi @grkhr @max-morrow

File: dbt_subprojects/dex/models/_projects/zeroex/zeroex_native_trades.sql [hunk 2]
```diff
@@ -21,7 +21,7 @@ FROM (
     {% for model in zeroex_models %}
     SELECT
       blockchain  as blockchain,
-      '0x API'  as project,
+      '0x-API'  as project,
```
Comment (jeff-dude) [inline dbt_subprojects/dex/models/_projects/zeroex/zeroex_native_trades.sql:24]: @RantumBits do you approve of this project name update? more detail in PR description. we are open to another value other than `-`, if you'd prefer (`0x_API`, `0xAPI`, etc.)

**Review by Hosuke — APPROVED** [review]

File: dbt_subprojects/dex/models/_projects/zeroex/zeroex_native_trades.sql (hunk 2 above)
Comment (RantumBits) [reply dbt_subprojects/dex/models/_projects/zeroex/zeroex_native_trades.sql:24]: 0x-API is good

**Comment by jeff-dude** [comment]
thank you all. we are finalizing analysis of downstream queries on dune which filter on `dex.trades` for these projects hardcoded and have ~100 queries to fix hardcoded value due to this change. we will merge this when we solve that.

**Review by couralex6 — APPROVED** [review]

**Pull request merged**